### PR TITLE
Per issue #53 adding CSV support

### PR DESCRIPTION
**Pull request by VDuda**
Not sure what else I need to modify to add support for csv as far as I can tell, I also need to make additional changes to the models.py and views.py 

I might need to perform a db migration as well? Please let me know.

### CODE REVIEW

**Comment by VDuda** [comment]
I believe the above should be sufficient, can you confirm? I just tested it on my end and it seems to work as expected.

File: pyas2/viewlib.py [hunk 1]
```diff
@@ -49,3 +49,6 @@ def indent_edifact(content):
 def indent_xml(content):
     this_xml = xml.dom.minidom.parseString(content)
     return this_xml.toprettyxml()
+
+def indent_csv(content):
```
Comment (abhishek-ram) [inline pyas2/viewlib.py]: This is not needed as  you are just returning the content

File: pyas2/views.py [hunk 2]
```diff
@@ -130,6 +130,8 @@ def get(self, request, pk, *args, **kwargs):
                     file_obj['content'] = viewlib.indent_edifact(file_obj['content'])
                 elif payload.content_type == 'application/XML':
                     file_obj['content'] = viewlib.indent_xml(file_obj['content'])
+                elif payload.content_type == 'text/CSV':
```
Comment (abhishek-ram) [inline pyas2/views.py]: This is not needed as  you are just returning the content

**Review by abhishek-ram — COMMENTED** [review]
Make the requested changes and I will merge

**Comment by abhishek-ram** [comment]
Thanks, I will include this in the next release.

**Pull request merged**